cluster-install/install.sh

```diff
@@ -2262,6 +2487,15 @@ step_collect_existing_trident_info() {
         logdebug "trident ACP image: not found"
     fi
 
+    # TORC imagePullSecrets
+    local -r torc_pull_secrets="$(echo "$torc_json" | jq -r '.spec.imagePullSecrets | join("\n")' 2> /dev/null)"
+    if [ -n "$torc_pull_secrets" ] && [ "$torc_pull_secrets" != "null" ]; then
+        logdebug "torc pull secrets:${__NEWLINE}$torc_pull_secrets"
+        _EXISTING_TORC_PULL_SECRETS="$torc_pull_secrets"
+    else
+        logdebug "torc pull secrets: not found"
+    fi
```

Comment:
Get the existing TORC pull secrets so we can later fiddle with it/determine whether we need to add the current IMAGE_PULL_SECRET to when generating upgrade patches. Probably could've made `_EXISTING_TORC_PULL_SECRETS` a boolean like "existing torc already has pull secret" or something, but I didn't think of that and I don't have time to fix it >.>